app/src/main/java/org/feature/fox/coffee_counter/ui/transaction/TransactionViewModel.kt

```diff
@@ -60,6 +76,51 @@ class TransactionViewModel @Inject constructor(
         transactions = response.data as MutableList<TransactionResponse>
     }
 
+    // reference: https://stackoverflow.com/questions/28232116/android-using-zxing-generate-qr-code
+    override suspend fun generateQRCode() {
```

Comment:
I think, if the user is sending money, his balance should be updated too/a new funding/purchase should be inserted into db.

Reply:
Resolved in code-week. Updates are done at other Views and therefore be fine with qr code